SemiTruck.cpp

```diff
@@ -0,0 +1,17 @@
+#include "SemiTruck.h"
+#include "Highway.h"
+
+SemiTruck::SemiTruck(const std::string& n) : Vehicle(n) {}
+
+void SemiTruck::closeWindows()
+{
+    setSpeed(65);
+    std::cout << name << ": closing the windows!" << std::endl;
+}
+
+void SemiTruck::tryToEvade()
+{
+    setSpeed(0);
+    std::cout << name << ": I'm stopping please don't arrest me!\n" << std::endl;
+}
```

Comment:
This function doesn't do any evasion.   It pulls the truck over.  
- [ ] rename the function to `pullOver`